plugins/chat/plugin.rb

```diff
@@ -19,6 +19,8 @@
 register_asset "stylesheets/common/chat-drawer.scss"
 register_asset "stylesheets/common/chat-index.scss"
 register_asset "stylesheets/mobile/chat-index.scss", :mobile
```

Comment:
maybe rename these too

Reply:
common/chat-index isnt full-page necessarily, cause it applies to all things
and the other one is mobile so full-page is implied and redundant

Reply:
Ok